extmod/modlwip.c

```diff
@@ -286,6 +286,15 @@ static const int error_lookup_table[] = {
 #define MOD_NETWORK_SOCK_DGRAM (2)
 #define MOD_NETWORK_SOCK_RAW (3)
 
+// Total queue length for UDP/raw connections.
```

Comment:
```suggestion
// Total queue length for buffered UDP/raw incoming packets.
```

(That's right, isn't it?)

Reply:
Yes, that's right.  Comment updated.